bottlerocket/agents/src/bin/eks-resource-agent/eks_provider.rs

```diff
@@ -31,9 +31,13 @@ use strum_macros::EnumString;
 /// The default region for the cluster.
 const DEFAULT_REGION: &str = "us-west-2";
 /// The default cluster version.
-const DEFAULT_VERSION: &str = "1.24";
+const DEFAULT_VERSION: &str = "1.32";
 const TEST_CLUSTER_CONFIG_PATH: &str = "/local/eksctl_config.yaml";
 const CLUSTER_CONFIG_PATH: &str = "/local/cluster_config.yaml";
+/// The default cluster managed node group values.
+const MNG_MIN_SIZE: i32 = 0;
+const MNG_MAX_SIZE: i32 = 2;
+const MNG_DESIRED_CAPACITY: i32 = 0;
```

Comment:
If we are setting this to 0, why are we creating it at all? Would it be better to fix the permissions error for adding taints than to set this to 0?